app/services/service_provider_logo_updater.rb

```diff
@@ -0,0 +1,123 @@
+require 'login_gov/hostdata'
+require 'subprocess'
+
+class ServiceProviderLogoUpdater
+  include ServiceProviderHelper
+
+  # :reek:TooManyStatements
+  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  def import_logos_to_active_storage
+    return unless Figaro.env.logo_upload_enabled
+
+    git_latest_idp_config
+    idp_config.each do |sp|
+      issuer = sp['issuer']
+      logo_name = sp['logo']
+      logger.info('~~ ' + issuer.to_s + ' logo: ' + logo_name.to_s)
```

Comment:
are we leaving this in intentionally?

Reply:
I think it's helpful for the person running the rake task to see that it's doing something. Could make some output that is less obnoxious?

Reply:
Yup, that's useful output. I use uncommon characters like `~` and `====` when doing test prints locally so I wanted to make sure this was left intentionally